File: Domain/Patient.cs
using System;

namespace Domain
{
    public class Patient
    {
        public Guid Id { get; set; }
        public string Name{ get; set; }
        public string PersonalId{ get; set; }
        public DateTime Date { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string Description { get; set; }
        public string Category { get; set; }
        public string City { get; set; }  
        public string Image { get; set; }  
    }
}

File: client-app/src/app/models/patient.ts
 export interface Patient{
        id: string;
        name: string;
        personalId: string,
        date: Date | null;
        dateOfBirth: Date | null,
        description: string;
        category: string;
        city: string;
    }

    export interface Photo {
        id: string;
        url: string;
        isMain: boolean;
    }